packages/tracing/test/integrations/node/postgres.test.ts

```diff
@@ -127,4 +131,23 @@ describe('setupOnce', () => {
 
     expect(loggerLogSpy).toBeCalledWith('Postgres Integration is skipped because of instrumenter configuration.');
   });
+
+  it('does not attempt resolution when module is passed directly', async () => {
+    const scope = new Scope();
+    jest.spyOn(scope, 'getSpan').mockReturnValueOnce(new Span());
+
+    new Integrations.Postgres({ module: mockModule }).setupOnce(
+      () => undefined,
+      () => new Hub(undefined, scope),
+    );
+
+    await new PgClient().query('SELECT NOW()', null);
+
+    expect(loadModule).not.toBeCalled();
+    expect(scope.getSpan).toBeCalled();
+  });
+
+  it('has valid module type', () => {
+    new Integrations.Postgres({ module: pg });
```

Comment:
can we add an assert on this? We can even do expect do not throw assertion.

Reply:
@AbhiPrasad done!